flink-connector-clickhouse/src/test/java/com/apache/flink/table/descriptors/ClickHouseTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apache.flink.table.descriptors;
+
+import org.junit.Test;
+
+import java.util.Map;
+
+public class ClickHouseTest {
+
+    @Test
+    public void toConnectorProperties() throws Exception {
+        ClickHouse clickhouse = new ClickHouse()
+            .version("1")
+            .address("jdbc:clickhouse://localhost:8123/default")
+            .batchSize(1)
+            .database("qtt")
+            .ignoreError(false)
+            .padding(2000L)
+            .table("insert_test")
+            .username("admin")
+            .password("admin")
+            .retryAttempts(3)
+            .retryInterval(3000L);
+        Map<String, String> connectorProperties = clickhouse.toConnectorProperties();
+        for (Map.Entry<String, String> entry : connectorProperties.entrySet()) {
+            System.out.println(entry.getKey() + ":" + entry.getValue());
```

Comment:
we should test that the map is what you expect and not print it to console

Reply:
@eskabetxe  thanks for your reply, related optimization has been completed, review again.